src/Combinatorics/Graphs.jl

```diff
@@ -552,6 +558,53 @@ function automorphisms(g::Graph{T}) where {T <: Union{Directed, Undirected}}
     result = Polymake.graph.automorphisms(pmg)
     return [[x+1 for x in a] for a in result]
 end
+
+
+@doc Markdown.doc"""
+    shortest_path_dijkstra(g::Graph{T}, s::Int64, t::Int64; reverse::Bool=false) where {T <: Union{Directed, Undirected}}
+
+Compute the shortest path between two vertices in a graph using Dijkstra's
+algorithm. All edges are set to have a length of 1. The optional parameter
+indicates whether the edges should be considered reversed.
+
+# Examples
+```jldoctest
+julia> g = Graph{Directed}(3);
+
+julia> add_edge!(g, 1, 2);
+
+julia> add_edge!(g, 2, 3);
+
+julia> add_edge!(g, 3, 1);
+
+julia> shortest_path_dijkstra(g, 3, 1)
+3-element Vector{Int64}:
+ 3
+ 2
+ 1
+
+julia> shortest_path_dijkstra(g, 1, 3)
+2-element Vector{Int64}:
+ 1
+ 3
```

Comment:
Both the arguments and the output seems reversed? 3 to 1 should give `3,1` and 1 to 3 should give `1, 2, 3`.